relayer/src/foreign_client.rs

```diff
@@ -188,6 +189,11 @@ pub fn build_update_client(
     dst_client_id: &ClientId,
     target_height: Height,
 ) -> Result<Vec<Any>, Error> {
+    // Wait for source chain to reach `target_height`
+    while src_chain.query_latest_height()? < target_height {
+        thread::sleep(Duration::from_millis(40))
```

Comment:
40ms between queries sounds a bit too short to me, but perhaps those are quicker than I expect?

Reply:
What would be a good value? Changed it to 100ms in the last commit. `query_latest_height()` takes 4msec on my machine with gaia nodes running locally. Also, since in this version (V0) nothing else is going on (we only have one blocking thread waiting here for source chain to advance) I don't think we take away time from anything in the relayer process.